content/community/conferences.md

```diff
@@ -12,6 +12,11 @@ Do you know of a local React.js conference? Add it here! (Please keep the list c
 
 ## Upcoming Conferences {#upcoming-conferences}
 
+### App.js Conf 2020 {#appjsonf2020}
+April 23 - 24, 2020 in Cracow, Poland
```

Comment:
Hmm, just interesting, earlier in this file the city was written differently:

> April 4-5, 2019 in **Kraków**, Poland

Which form is correct?

Reply:
Actually, both of them. One is the original Polish spelling and the second one is official English translation.

I will change it to keep it consistent.

Reply:
@lex111 Fixed it.